libs/kotlin.json

```diff
@@ -27,6 +27,7 @@
   {"id":"kt.konsent","imports":["com.oneeyedmen.konsent"],"name":"Konsent","repo":"dmcg/konsent","tags":["acceptance"],"tech":["eng-tools"],"status":"model-ready"},
   {"id":"kt.kotlinpoet","imports":["com.squareup.kotlinpoet."],"name":"Anko","repo":"square/kotlinpoet","tags":["code-generators"],"tech":["eng-tools"],"status":"awaiting-model"},
   {"id":"kt.advent-kt","imports":["com.github.vassilibykov.adventkt"],"name":"AdventKT","repo":"vassilibykov/AdventKT","tags":["text-adventures"],"tech":["graphics-gaming"],"status":"model-ready"},
+  {"id":"kt.coil","imports":["coil"],"name":"coil","repo":"coil-kt/coil","tags":["android-images"],"tech":["graphics-gaming"],"status":"model-ready"},
```

Comment:
for new libraries, status is always "awaiting-model". We change it to "model-ready" when, well, when the model is ready.

Reply:
done